src/Pages/hth/LoginHTH/LoginHTH.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import "./LoginHTH.scss";
+
+class LoginHTH extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      pw: "",
+      noticeText: "",
+    };
+  }
+
+  onKeyPressHandle = (event) => {
+    if (event.key === "Enter") {
+      if (this.state.id.includes("@") && this.state.pw.length > 4)
+        this.onClickHandle();
+    }
+  };
```

Comment:
```suggestion
  onKeyPressHandle = (event) => {
    if (event.key !== "Enter") return 
    const { id, pw } = this.state
    const isValid = id.includes("@") && pw.length > 4

    if (isValid) this.onClickHandle();
  };
```
- early return을 사용하면 if 문의 depth를 낮출 수 있어 유용합니다
- 구조분해 할당 적절히 이용해주세요
- 조건문에 대해서 적당한 변수명을 붙여두면 가독성을 높일 수 있습니다.
- 준식님 리뷰에 맞게 함수 이름 바꿔주세요.